app/src/pages/inside/projectSettingsPageContainer/content/notifications/notifications.jsx

```diff
@@ -172,7 +172,7 @@ export const Notifications = ({ setHeaderTitleNode }) => {
     }
 
     return () => setHeaderTitleNode(null);
-  });
+  }, [setHeaderTitleNode]);
```

Comment:
pls read the doc https://reactjs.org/docs/hooks-reference.html#usestate
I guess we should be dependent at least on `notifications` array